data-egress.tf

```diff
@@ -231,6 +231,26 @@ resource "aws_dynamodb_table_item" "housing_SAS_data_egress_config" {
   ITEM
 }
 
+resource "aws_dynamodb_table_item" "data_warehouse_config" {
+  table_name = aws_dynamodb_table.data_egress.name
+  hash_key   = aws_dynamodb_table.data_egress.hash_key
+  range_key  = aws_dynamodb_table.data_egress.range_key
+
+  item = <<ITEM
+  {
+    "source_prefix":                {"S":    "dwh/zip/*"},
+    "pipeline_name":                {"S":    "DWh-Transform-Json"},
+    "recipient_name":               {"S":    "DataWarehouse"},
+    "transfer_type":                {"S":    "SFT"},
+    "source_bucket":                {"S":    "${data.terraform_remote_state.common.outputs.published_bucket.id}"},
+    "destination_prefix":           {"S":    "/data-egress/warehouse/"},
```

Comment:
is this the agreed prefix? I thought all SFT egress was to the data warehouse through the data-egress service - this seems too generic